src/main/java/in/pune/royforge/eledgerapi/data/dao/WalletDAOImpl.java

```diff
@@ -201,4 +191,24 @@ private void setWalletData(WalletEntity walletEntity, WalletData walletData) {
 		walletData.setUpdatedDate(walletEntity.getUpdatedDate());
 		walletData.setWalletId(walletEntity.getWalletId());
 	}
+	
+	// By taking input {lenderId} to delete the wallet.
+	@Override
+	public boolean deleteWallet(long walletId) {
+		Optional<WalletEntity> existedWallet = walletEntityRepository.findById(walletId);
+		if (existedWallet.isPresent()) {
+			WalletEntity walletEntity = new WalletEntity();
+			walletEntity.setCreatedDate(existedWallet.get().getCreatedDate());
```

Comment:
from line 201 to 206, create a separate method and use it here and in fetch API also.